resources/cards/models/card_editable.yml

```diff
@@ -0,0 +1,23 @@
+CardEditable:
+  title: CardEditable
+  required:
+  - description
+  - front
+  type: object
+  properties:
+    description:
+      title: Description
+      maxLength: 255
+      type: string
+    front:
```

Comment:
I don't think front and back are editable fields for a card

Reply:
correct